search-service/src/main/kotlin/com/egm/stellio/search/authorization/EntityAccessRightsService.kt

```diff
@@ -321,6 +321,22 @@ class EntityAccessRightsService(
             }
     }
 
+    suspend fun getEntitiesIdsOwnedBySubject(
```

Comment:
Why not directly do a deleteEntitiesOwnedBySubject?

Reply:
the deletion doesn't happen unless you recieve an `EntityDeleteEvent` in `IAMListener`. in `IAMListener`  the user (`SubjectReferential` for better terms) is deleted then the entities created by that user are deleted.
We need to first get those entities using the user's access rights. 
This way the code is better organized with every service doing his part of the whole process and _getEntitiesIdsOwnedBySubject()_ can be used in other places.

Reply:
Agree